test/testapps/RazorIdeTesting/Controllers/HomeController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
```

Comment:
Ignore everything below here, I checked in a project that I created from the template.